apps/backend/core/configuration/services.py

```
@@ -0,0 +1,278 @@
+            "openai_api_key",
+            "anthropic_api_key",
+            "langchain_api_key",
+            "supabase_service_role_key",
+            "pusher_secret",
+            "clerk_secret_key",
+            "roblox_api_key",
+            "sentry_dsn",
+        ]
+
+        for field in sensitive_fields:
+            if field in data and data[field]:
```

Ruff (0.14.4)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)